implementation/contracts/deposit/DepositUtils.sol

```diff
@@ -116,7 +116,7 @@ library DepositUtils {
     }
 
     /// @notice                 Syntactically check an SPV proof for a bitcoin transaction with its hash (ID).
-    /// @dev                    Stateless SPV Proof verification documented elsewhere (see github.com/summa-tx/bitcoin-spv).
+    /// @dev                    Stateless SPV Proof verification documented elsewhere (see `github.com/summa-tx/bitcoin-spv`).
```

Comment:
Don't think we need backticks here; without them, most processors of the text will link it. May even want to drop the backticks + add an `https://` before it.